.github/scripts/locales-cli-editor/main.js

```diff
@@ -0,0 +1,307 @@
+/*
+ * SPDX-FileCopyrightText: 2025 Pagefault Game
+ * SPDX-FileContributor: SirzBenjie
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+/*
+ * Interactive CLI to reword or delete locales keys for the "en" file.
+ * Usage: `node scripts/manage-locales.js`
+ */
+
+import fs from "node:fs";
+import path from "node:path";
+import chalk from "chalk";
+import inquirer from "inquirer";
+import { dirname, resolve } from "node:path";
+import { fileURLToPath } from "node:url";
+
+//#region Constants
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+/** The path to the root of the locales directory */
+const LOCALES_DIR = resolve(__dirname, "../../../");
+/** The language code to edit */
+const LANG_CODE = "en";
+/** */
+const LOCALES_PATH = path.join(LOCALES_DIR, LANG_CODE);
+/**
+ * The list of supported languages
+ * @type {readonly string[]}
+ */
+const SUPPORTED_LANGS = fs
+  .readdirSync(LOCALES_DIR)
+  .filter(f => f !== "en" && fs.statSync(path.join(LOCALES_DIR, f)).isDirectory());
+/**
+ * The list of JSON files that can be edited
+ * @type readonly string[]
```

Comment:
```suggestion
 * @type {readonly string[]}
```
I think the `{}` is required?